pkgs/love2d/example/assets/love_example_browser/examples/150_shaders.lua

```
@@ -0,0 +1,51 @@
+			number luma = dot(vec3(0.299f, 0.587f, 0.114f), color.rgb);
+			return mix(color, tint * luma, strength);
+		}
+	]])
+
+	--Send an initial value for tint to the shader in the form of a table
+    shader:send('tint', {1.0,1.0,1.0,1.0})    
+end
+
+function love.update()
+	--Update the strength of the shader with the height of the mouse y position
+	local x,y = love.mouse.getPosition()
```

Luacheck (1.2.0)

[warning] unused variable 'x'

(W211)